lib/oli/delivery.ex

```diff
@@ -161,7 +161,6 @@ defmodule Oli.Delivery do
       enroll(user.id, section.id, lti_params)
 
       {:ok, updated_section} = maybe_update_section_contains_explorations(section)
-      {:ok, _} = Sections.update_resource_to_container_map(section)
```

Comment:
Do we want to pay the cost of warming the cache on product/section creation or first student access?